app/src/main/java/org/jellyfin/mobile/settings/SettingsFragment.kt

```diff
@@ -164,6 +168,13 @@ class SettingsFragment : Fragment() {
             titleRes = R.string.external_player_app
             enabled = appPreferences.videoPlayerType == VideoPlayerType.EXTERNAL_PLAYER
         }
+        pref(Constants.PREF_SUBTITLE_STYLE) {
+            titleRes = R.string.pref_subtitle_style
+            summaryRes = R.string.pref_subtitle_style_summary
+            defaultOnClick {
+                startActivity(Intent(Settings.ACTION_CAPTIONING_SETTINGS))
```

Comment:
Do we need to check if the system exposes these settings. IIRC in some Android flavors the subtitle settings are not accessible.

> In some cases, a matching Activity may not exist, so ensure you safeguard against this.
> - https://developer.android.com/reference/android/provider/Settings#ACTION_CAPTIONING_SETTINGS

Reply:
You're right, a try catch + Toast would be a good idea in this case.

Reply:
Maybe just check if the intent is available and hide/disable the option